flamingpy/codes/surface_code.py

```diff
@@ -130,6 +133,9 @@ def RHG_graph(
 
             where each b above can be 'primal', 'dual', or 'periodic'.
             By default, 'primal'  is used (i.e. ['primal', 'primal', 'primal']).
+            Note that in the above boundaries, we consider the z-axis to have
+            the role of temporal dimension. This is relevant for quantum memory
+            simulations and gives the temporal boundaries a special role.
```

Comment:
This should be in the docstring to SurfaceCode, not RHG_graph. I would suggest:

> For the "open" and "toric" boundary choice, we imagine qubits are encoded into the x-y plane and propagated in time (which we associate with the z-axis).

Reply:
Maybe it makes more sense to put your suggestion in @sduquemesa's PR #62? Here we don't have the toric code defined yet. I'll move this docstring from RHG_graph to SurfaceCode anyway and we can see what the final docstring should be when merging both.

Reply:
Anything I should do from this comment?

Reply:
I will take care of this tomorrow. I'll just make a small commit to your branch to update the docstring @sduquemesa

Reply:
done ✔️